### PR TITLE
added the italian translation for unit 1 chapter5 issue #45

### PR DESCRIPTION
**Pull request by Nolanogenn**

### CODE REVIEW

File: chapters/it/chapter5/2.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# What if my dataset isn't on the Hub?
```
Comment (lewtun) [inline chapters/it/chapter5/2.mdx:1]: Should this be translated?

File: chapters/it/chapter5/1.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Introduction
```
Comment (lewtun) [inline chapters/it/chapter5/1.mdx:1]: Should this be translated?

**Review by lewtun — COMMENTED** [review]
Thank you for this contribution 🤗 ! We just need to add the relevant sections to the `_toctree.yml` file, i.e. please add this:

```yaml
- title: 5. The 🤗 Datasets library # Translate this
  sections:
  - local: chapter5/1
    title: Introduction # Translate this
  - local: chapter5/2
    title: What if my dataset isn't on the Hub? # Translate this
```

File: chapters/it/chapter5/2.mdx (hunk 1 above)
Comment (Nolanogenn) [reply chapters/it/chapter5/2.mdx]: Fixed in latest commit

File: chapters/it/chapter5/1.mdx (hunk 2 above)
Comment (Nolanogenn) [reply chapters/it/chapter5/1.mdx]: Fixed in latest commit

**Comment by lewtun** [comment]
Hey @Nolanogenn just checking if you need any help on adding the `_toctree.yml` file to this PR?

**Comment by Nolanogenn** [comment]
> Hey @Nolanogenn just checking if you need any help on adding the `_toctree.yml` file to this PR?

**Pull request closed without merging**